content/v1.20/guides/metrics.md

```diff
@@ -23,36 +23,91 @@ prometheus.io/port: "8080"
 prometheus.io/scrape: "true"
 ```    
 
+## Crossplane core metrics
```

Comment:
come to think of it, i don't think we should update old versions like `v1.20` and `v2.0-preview`, because a lot of these values had different names or didn't even exist in those previous versions.  but let's also try not to overcomplicate it either, so what do you think of the following?

* don't bother updating `v2.0-preview`, that release is a bit of a one-off and those docs aren't even exposed in the version picker anymore
* don't bother updating `v1.20`, unless you really want to get every old name correct for `crossplane_composition_watches_*` etc. as shown in https://github.com/crossplane/crossplane/releases/tag/v2.0.0
* remove `circuit_breaker_*` metrics from `v2.0`, those were added in `v2.1`
* keep `v2.1` and `master` as is 🎉

Reply:
Great points!

- `v2.0-preview` changes i threw away
- put an effort to get every `v1.20` old name correct -> we already went too far for shortcuts 😅 
- `circuit_breaker_*` was removed from everywhere except `v2.1`
- `v2.1` master kept as is ;)